controller/types/types.go

```diff
@@ -89,17 +89,34 @@ type Key struct {
 }
 
 type Job struct {
-	ID        string            `json:"id,omitempty"`
-	AppID     string            `json:"app,omitempty"`
-	ReleaseID string            `json:"release,omitempty"`
-	Type      string            `json:"type,omitempty"`
-	State     string            `json:"state,omitempty"`
-	Cmd       []string          `json:"cmd,omitempty"`
-	Meta      map[string]string `json:"meta,omitempty"`
-	CreatedAt *time.Time        `json:"created_at,omitempty"`
-	UpdatedAt *time.Time        `json:"updated_at,omitempty"`
+	ID         string            `json:"id,omitempty"`
+	AppID      string            `json:"app,omitempty"`
+	ReleaseID  string            `json:"release,omitempty"`
+	Type       string            `json:"type,omitempty"`
+	State      JobState          `json:"state,omitempty"`
+	Cmd        []string          `json:"cmd,omitempty"`
+	Meta       map[string]string `json:"meta,omitempty"`
+	ExitStatus int32             `json:"exit_status"`
```

Comment:
What do you think about making this a pointer so the JSON output of jobs in a non-down state don't include `"exit_status": 0`?